File: Java/ProgramAdviser/app/src/main/java/bjfu/it/zhangsixuan/programadviser/ProgramExpert.java
package bjfu.it.zhangsixuan.programadviser;

public class ProgramExpert {
    public String getLanguage(String feature) {
        String res;
        switch (feature) {
            case "fast":
                res = "C/C++";
                break;
            case "easy":
                res = "Python";
                break;
            case "new":
                res = "Kotlin";
                break;
            case "OO":
                res = "Java";
                break;
            default:
                res = "U got me";
        }
        return res;
    }
}

File: Kotlin/ProgramAdviserK/app/src/main/java/bjfu/it/zhangsixuan/programadviserk/ProgramExpert.kt
package bjfu.it.zhangsixuan.programadviserk

class ProgramExpert {
    fun getLanguage(feature: String): String =
        when (feature) {
            "fast" -> "C/C++"
            "easy" -> "Python"
            "new" -> "Kotlin"
            "OO" -> "Java"
            else -> "You got me."
        }
}